docs/src/test/java/io/quarkus/docs/vale/ValeAsciidocLint.java

```diff
@@ -154,7 +154,10 @@ public Map<String, ChecksBySeverity> lintFiles() throws Exception {
 
         DockerImageName valeImage = DockerImageName.parse(imageName);
 
-        List<String> command = new ArrayList<>(List.of("--config=/vale/vale.ini",
+        // --config=vale.ini is not really necessary as it is the default
+        // location, but using it here explicitly to have the test case fail
+        // if the file is missing.
+        List<String> command = new ArrayList<>(List.of("--config=vale.ini",
```

Comment:
Doesn't this file name begin with a `.`?

```suggestion
        List<String> command = new ArrayList<>(List.of("--config=.vale.ini",
```